api/src/test/java/org/openmrs/api/context/TestUsernameAuthenticationScheme.java

```diff
@@ -12,16 +12,16 @@
 import org.openmrs.User;
 
 /*
- * This test auth scheme logs a user simply based on asserting that its UUID is valid and points to an exisiting user.
+ * This test auth scheme logs a user simply based on asserting that its username is valid and points to an existing user.
  */
-public class TestUuidAuthenticationScheme extends DaoAuthenticationScheme {
+public class TestUsernameAuthenticationScheme extends DaoAuthenticationScheme {
 
 	@Override
 	public Authenticated authenticate(Credentials credentials) throws ContextAuthenticationException {
 		
-		TestUuidCredentials creds = (TestUuidCredentials) credentials;
+		TestUsernameCredentials creds = (TestUsernameCredentials) credentials;
```

Comment:
I do not see a need for this casting. I thought you would just do credentials.getClientName() on line 24 below and as a result, you do not need the skipAuthenticationInBaseSetup changes in BaseContextSensitiveTest.java